nf_core/module-template/tests/test.yml

```
@@ -1,6 +1,6 @@
 ## TODO nf-core: Please run the following command to build this file:
 #                nf-core modules create-test-yml {{ tool }}/{{ subtool }}
-- name: {{ tool }} {{ subtool }}
+- name: {{ tool }} {{ subtool if subtool != None else '' }}
```

I think that this might break the YAML linting if no subtool due to trailing whitespace?

```suggestion
- name: {{ tool }}{{ ' '+subtool if subtool else '' }}
```